lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,38 @@
 module Minesweeper_2pl
   class App
-    def call
-      "Hello World"
+    SIZE = 100
+    BOMB_COUNT = 10
+    attr_accessor :game, :cli
+
+    def start
+      self.setup
+      self.play_game
+    end
+
+    def setup
+      game = Game.new
+      cli = CLI.new
+      self.cli = cli
+      self.game = game
+      self.game.setup(SIZE, BOMB_COUNT)
+    end
+
+    def play_game
+      while self.game.game_over != true
+        self.game.print_board
+        move = nil
+        while move == nil
+          self.cli.ask_for_move
+          move = self.cli.get_player_input(self.game)
```

Comment:
1) What is the difference between `ask_for_move` and `get_player_input`?
2) Why does `get_player_input` need access to the game?
3) It's a great direction to extract the responsibility for getting input to a separate object like you've done here. But the name `CLI` seems unnecessarily coupled to the command line. What if we were asking for player input from some other interface, like a web browser (where this project is ultimately headed!) or a voice interface? Is there a way to depend on an _abstraction_ responsible for "getting player input" rather than a concrete implementation like "command line interface"?